template/_package.json

```diff
@@ -3,8 +3,14 @@
   "version": "1.0.0",
   "description": "<%= description %>",
   "author": "<%= author %>",
-  "private": true,
-  "scripts": {<% if (server === 'adonis') { %>
+  "private": true, <% if (server === 'feathers') { %>
+  "directories": {
+    "lib": "server",
+    "config": "server/config",
+    "test": "server/test"
+  }, <% } %>
+  "scripts": {<% if (server === 'feathers') { %>
+    "feathers": "feathers",<% } %><% if (server === 'adonis') { %>
```

Comment:
Should be `else if`

Reply:
placing `else if` causes non execution of `else` block immediately after `if (server === 'adonis')`
resulting to non generation of `dev, build, start, generate` tasks

should I place the `feathers` task as last task (after `test`)?

Reply:
Oh, I gotcha. Yes, putting `if (server === 'feathers')` after `dev, build, start, generate` makes more sense

Reply:
Fixed in commit 1910cb0da92dbcbb76618c19b61afb78bc0682d2